examples/experimental/bert_squad_pytorch/data.py

```diff
@@ -0,0 +1,30 @@
+import os
+import constants
+import torch
+from transformers.data.processors.squad import SquadV1Processor
+from transformers import squad_convert_examples_to_features
+
+def load_and_cache_examples(data_dir: str, config, model_type, max_seq_length, doc_stride, max_query_length, model_name_or_path, evaluate=False, output_examples=False):
+    config_class, tokenizer_class, model_class = constants.MODEL_CLASSES[model_type]
+    tokenizer = tokenizer_class.from_pretrained(
+        config["model_name_or_path"], do_lower_case=True, cache_dir=None
+    )
+    try:
+        import tensorflow_datasets as tfds
+    except ImportError:
+        raise ImportError("Tensorflow_datasets needs to be installed.")
```

Comment:
blocking: I don't think this `try`, `except` block makes much sense in a Determined example, since we're explicitly calling out the dependencies in `startup-hook.sh`. I would move remove the try-except block and move this import with the other imports.